src/components/Call.vue

```diff
@@ -201,11 +208,12 @@ export default {
     },
     // leave call, destroy call object, and reset local state values
     leaveAndCleanUp() {
+      if (!this.callObject) return;
       if (this.screen?.local) {
         this.callObject.stopScreenShare();
       }
       this.callObject.leave().then(() => {
-        this.callObject.destroy();
+        this.callObject?.destroy();
```

Comment:
For my own curiosity, what would make this falsy? I don't see anything resetting the call object in this code, is this a Vue thing?